BiCo/src/app/app-routing.module.ts

```diff
@@ -11,6 +11,8 @@
 import { LoginComponent } from './components/login/login.component';
 import { BuscadorNegocioComponent } from './components/buscador-negocio/buscador-negocio.component';
 import { VerReservasComponent } from './components/reservas/ver-reservas/ver-reservas.component';
+import { MisNegociosComponent } from './components/negocios/mis-negocios/mis-negocios.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-2021/BiCo-Frontend/pullRequest?prid=7237503)
```suggestion
import { MisNegociosComponent } from "./components/negocios/mis-negocios/mis-negocios.component";
```